### DISCUSSION TITLE
Beta cores are now the default option

### DISCUSSION THREAD

**Discussion by ethanaobrien** [EmulatorJS/EmulatorJS#305] [General]
Hello everyone, I am excited to announce that the beta option is no longer available, for beta cores are now the default option!

Some labels may be messed up in the controller settings (nintendo 64 had one, but its fixed) open an issue if any buttons are not working. This is because I have updated retroarch to a newer version and the default inputs have changed.

If you want to continue to use the old cores, you can set the variable `EJS_oldCores` to true. This will load the old cores.

The new cores are currently available for beta systems. More systems coming soon! The emulator will use the old version if the new version is not yet supported.

For those people who like statistics: the new cores use anywhere from 1/10th to 1/2th of the amount of cpu of the old versions (that means it runs better)

Thank you all for using my product! I really enjoy working on this and you guys giving me ideas is 90% of my work!

What do y'all have to say? Leave a comment.

**Comment by Wade7wastaken** [comment]
I tried out the new cores, and I have to say, they are much better. The only problem I found is that n64 savestates don't work at all. Manually downloading and reuploading a state doesn't work, choosing `keep in browser` doesn't work, and the savestate hotkeys don't work either. Other than that, all the other cores worked just fine.

**Reply by ethanaobrien** [reply]
Thanks for the find! I’m going to look into this

**Reply by ethanaobrien** [reply]
This is fixed!

**Comment by Ramaerel** [comment]
These new cores and the updates you have been pushing are amazing, thank you for all your hard work! 
Just a heads up, the front page should probably reflect using EJS_oldCores instead of EJS_BETA. 

Also, do you have any plans for more features at this point in time?

**Reply by ethanaobrien** [reply]
Thanks! I really enjoy working on this project! Ever since I finally cracked simulate input my development has been flying.

>Just a heads up, the front page should probably reflect using EJS_oldCores instead of EJS_BETA.

Ah yes, thanks for the reminder. I'll fix this tomorrow.

>Also, do you have any plans for more features at this point in time?

Not really, I plan to go through and fix all the issues I can.

I have also been thinking about removing netplay at least for the moment, completely stripping the feature out. It's extremely incompatible and there's this weird bug specifically for edge when the netplay option is even available. Do you have an opinion on this?

**Reply by Ramaerel** [reply]
While the netplay is cool, I think you would be better off stripping it out for now. Making sure its all working properly and you're content with it is likely more important. Besides, building netplay onto your new cores could cause issues right? From what I can tell the retroarch netplay doesn't work with the emscripten cores, so you'd need to build it from the ground up

**Reply by ethanaobrien** [reply]
Yeah there are bound to be issues with the new cores. I do think I should strip it out for now and build it back up. I’ll do that when I can
